synapse/handlers/room_member.py

```diff
@@ -261,6 +261,7 @@ async def _local_membership_update(
         target: UserID,
         room_id: str,
         membership: str,
+        origin_server_ts: Optional[int],
```

Comment:
It looks like there's a missing `= None` here?
```suggestion
        origin_server_ts: Optional[int] = None,
```